Userland/Libraries/LibCore/Platform/MachMessageTypes.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024, Andrew Kaster <akaster@serenityos.org>
+ * Copyright (c) 2024, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Platform.h>
+
+#if !defined(AK_OS_MACH)
+#    error "This file is only available on Mach platforms"
+#endif
+
+#include <mach/mach.h>
+
+namespace Core::Platform {
```

Comment:
These message ports are now even more Ladybird-specific. Keeping them in LibCore feels like the wrong layer. Should they be moved to LibWebView/LibWeb?

Reply:
hmm, I caught myself thinking the same, but then found RequestServer needs `register_with_mach_server` which also pulls message type and definition, and it would be wrong to include LibWebView/LibWeb from there.